samples/crypto/psa_tls/sample.yaml

```diff
@@ -108,6 +145,30 @@ tests:
       - nrf54l15dk/nrf54l15/cpuapp
       - nrf54l15pdk/nrf54l15/cpuapp
     tags: ci_build oberon sysbuild ci_samples_crypto
+  sample.psa_tls.1_3_server.ecdsa.oberon:
```

Comment:
Maybe you meant `...sample.psa_tls.tls_1_3_server..` (that's what I see in the previous sample)

Reply:
It seems that the paths for 54l15 samples were made different, I just followed the naming scheme from before: `tls_server` -> `tls_1_3_server` and `server` -> `1_3_server`